front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,188 @@
+by allowing to save the show's latest season and episode watched*/
+const ProgressData = ({ season, episode }) => {
+  const refSeason = useRef();
+  const refEpisode = useRef();
+  const saveProgressData = (() => {
+    //the function will be further developed in sprint 2
+    let progress = [];
+    var season = React.findDOMNode(this.refs.season).value;
+    var episode = React.findDOMNode(this.refs.episode).value;
+    progress.push(season);
+    progress.push(episode);
+  });
+  return (
+    <form onSubmit={(e) => saveProgressData()} >
+      <label for="season">Current Season: </label>
+      <input id="season" class="progress" value={season} ref={refSeason} />
+      <br /> <br />
+      <label for="episode">Current Episode:</label>
+      <input id="episode" class="progress" value={episode} ref={refEpisode} />
+      <br /><br />
+      <input type="submit" value="Save Progress" />
+    </form>
+  )
+}
+
+/*the component displays and stores user's chosen platform*/
+
+const PlatformData = () => {
+
+  const refNetflix = useRef();
+  const refPrime = useRef();
+  const refHulu = useRef();
+  const refApple = useRef();
+
+
+  const savePlatform = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div>
+      <p>Select the platform: </p>
+      <form onSubmit={(e) => savePlatform()}>
+
+        <input type="checkbox" id="netflix" value="Netflix" ref={refNetflix} />
+        <label for="netflix">Netflix  </label>
+        <input type="checkbox" id="prime" value="Prime" ref={refPrime} />
+        <label for="prime">Prime  </label>
+        <input type="checkbox" id="hulu" value="Hulu" ref={refHulu} />
+        <label for="hulu">Hulu  </label>
+        <input type="checkbox" id="apple" value="Apple" ref={refApple} />
+        <label for="apple">Apple TV+  </label><br /><br />
+        <input type="submit" value="Save Platform" />
+      </form>
+    </div>
+  )
+}
+
+/*the component displays and stores general info about the show*/
+
+const Description = ({ genre, description, totalEpisodes, isMovieN }) => {
+  const refDescription = useRef();
+  const refIsMovie = useRef();
+  const refGenre = useRef();
+  const refTotalEpisodes = useRef();
+  let movieV = { isMovieN };
+  if (movieV) {
+    movieV = "Yes";
+  } else {
+    movieV = "No";
+  }
+  const saveMovieData = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <span id="genre" ref={refGenre}>{genre}</span><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="totalEpisodes">Total Episodes: </label>
+      <span id="totalEpisodes" ref={refTotalEpisodes}>{totalEpisodes}</span><br /><br />
+      <label for="movie">Is it a movie? </label>
```

Comment:
The user should, generally, not be the one setting whether something is a movie. That'll be information we get from the Trakt API, which has a media type variable. I believe our mockaroo api currently has an isMovie boolean for each show.

Reply:
yes, the value of the isMovie variable is from API; in my earlier commit there was a bug, but I fixed it in the newest commit